src/pages/diagnose/webhooks.md

```diff
@@ -56,4 +56,8 @@ Slack supports integrating directly with webhooks.
 1. Enable incoming webhooks for your Slack instance (Tutorial [here](https://api.slack.com/messaging/webhooks#enable_webhooks))
 2. Get a hooks.slack.com address for your channel (Tutorial [here](https://api.slack.com/messaging/webhooks#create_a_webhook))
 3. Open up Railway, navigate to your project. Under Deployments -> Settings -> Webhooks, paste your URL
-4. Click the checkmark to save
\ No newline at end of file
+4. Click the checkmark to save
+
+## Crash Notifications
+
+Railway projects now allow you to pipe out project crashes via the configured webhook.
```

Comment:
I'm unsure if we necessarily need this section, especially cause the events we emit might change later

```suggestion
Railway projects now allows you to pipe out project crashes via the configured webhook.
```